src/pkg/caendr/caendr/models/run/gcp.py

```diff
@@ -158,7 +158,7 @@ def split_job_name(cls, kind, job_name) -> (str, str):
     # Loop through valid database operations
     for op in DbOp:
       if data_id == make_dns_name_safe(op.name):
-        return _kind, op.name
+        return make_dns_name_safe(kind), op.name
```

Comment:
are we checking for overlapping values returned by make_dns_name_safe()? For instance, if someone submits `A-A` and someone else submits `A_A` will these map to the same output value?